.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 cmd/globalcv/env.sh
 cmd/globalcv/globalcv
+routes.md
```

Comment:
why are you ignoring routes.md?

Reply:
temporarily since I just added the contents to README